metrics/src/main/scala/org/broadinstitute/dsde/rawls/metrics/Expansion.scala

```diff
@@ -44,6 +45,34 @@ object Expansion {
     */
   implicit def RawlsEnumerationExpansion[A <: RawlsEnumeration[_]] = new Expansion[A] {}
 
+  /**
+    * Implicit expansion for HttpMethod.
+    */
+  implicit object HttpMethodExpansion extends Expansion[HttpMethod] {
```

Comment:
Can you add cases to `ExpansionSpec` for this (`HttpMethod`, `Uri`, `HttpStatus`)?